src/setupTests.ts

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import "@testing-library/jest-dom";
```

Comment:
setupTests runs before every single test file. If you look in the jest config file above, you can see where we tell jest to do use this file for this purpose.